.github/workflows/linters.yml

```diff
@@ -32,20 +32,6 @@ jobs:
           [ -f .hintrc ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css-js/.hintrc
       - name: Webhint Report
         run: npx hint --telemetry=off .
-  stylelint:
-    name: Stylelint
-    runs-on: ubuntu-18.04
-    steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-node@v1
-        with:
-          node-version: "12.x"
-      - name: Setup Stylelint
-        run: |
-          npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
-          [ -f .stylelintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css-js/.stylelintrc.json
-      - name: Stylelint Report
-        run: npx stylelint "**/*.{css,scss}"
   eslint:
     name: ESLint
     runs-on: ubuntu-18.04
```

Comment:
- Kindly enable your stylelint from your [linters.](https://github.com/microverseinc/linters-config/tree/master/html-css-js).
- Fix your MIT License link.
- Your package.json file it's missing the scripts for build and run the server.

Reply:
Hello @carlos-ssh,
Thank you for your feedback. 
We have fixed the issues.

Reply:
This issue is well resolved! ✔️